.ci/install_crio.sh

```diff
@@ -32,6 +32,13 @@ crio_repo=$(get_version "externals.crio.url")
 crio_repo="${crio_repo#*//}"
 crio_config_file="/etc/crio/crio.conf"
 
+# Remove CRI-O repository if already exists on Fedora
+if [ "$ID" == "fedora" ]; then
+	if [ -d "${GOPATH}/src/${crio_repo}" ]; then
+		sudo rm -rf "${GOPATH}/src/${crio_repo}"
```

Comment:
You could make this `rm -r` as the `-f` isn't required due to the `if` test above.